sanitizer/PeliasParameterError.js

```diff
@@ -0,0 +1,15 @@
+class PeliasParameterError extends Error {
+  constructor(message = '') {
+    super(message);
```

Comment:
Can message be either a string or type of error?

Reply:
It can only be a string. However it might make sense for other parts of the code to detect an attempt to stuff an error object in instead, and fix that. I will add a corresponding test.

Reply:
A test for the aforementioned case has now been added